src/rust/engine/fs/store/src/cli_options.rs

```diff
@@ -65,6 +64,87 @@ pub struct StoreCliOpt {
 }
 
 impl StoreCliOpt {
+    pub fn new_local_only(local_store_path: PathBuf) -> Self {
+        Self {
+            local_store_path: Some(local_store_path),
+            cas_server: None,
+            remote_instance_name: None,
+            cas_root_ca_cert_file: None,
+            cas_client_certs_file: None,
+            cas_client_key_file: None,
+            cas_oauth_bearer_token_path: None,
+            upload_chunk_bytes: 0,
+            store_rpc_retries: 0,
+            store_rpc_concurrency: 0,
+            store_batch_api_size_limit: 0,
+            header: vec![],
+        }
+    }
+
+    pub fn to_cli_args(&self) -> Vec<OsString> {
+        let mut ret: Vec<OsString> = vec![];
+        if let Some(local_store_path) = &self.local_store_path {
```

Comment:
Would a helper function like this make this function nicer to look at (to avoid all the explicit `.into()` calls)?:

```
fn push_arg<S: Into<OsString>>(args: &mut Vec<OsString>, next_arg: S) {
    args.push(arg.into());
}
```

Also, just a suggestion and not required.

Reply:
Good idea! Done.